examples/jsm/utils/SceneUtils.js

```diff
@@ -118,8 +118,55 @@ function createMultiMaterialObject( geometry, materials ) {
 
 }
 
+function reduceVertices( func, initialValue ) {
+
+	let value = initialValue;
+	const vertex = new Vector3();
+
+	this.traverseVisible( ( object ) => {
```

Comment:
Since `reduceVertices()` is no method of `Object3D` anymore, `this` does not point to the right spot here. I guess the function needs an additional parameter representing the 3D object.

Reply:
Oh, good call. I was also asked to add this to an example; I'll do that too to make sure I don't miss any other errors.

Reply:
I've resolved the merge conflicts and updated `reduceVertices()` so it works now in context of `SceneUtils`. The function has now three instead of two parameters.